sos/report/plugins/__init__.py

```diff
@@ -1674,6 +1685,9 @@ def add_copy_spec(self, copyspecs, sizelimit=None, maxage=None,
         if self.get_option('since'):
             since = self.get_option('since')
 
+        # User's plugin option argument takes precedence over plugin passed arg
+        maxage = self.get_option('maxage') or maxage
+
```

Comment:
Shall not we unify the code here? Both manipulations perform the same at the end, the unification would improve clarity of code.